packages/components/src/Typeahead/Typeahead.scss

```diff
@@ -93,8 +93,9 @@ $tc-typeahead-items-font-size: 1.5rem !default;
 
 		.icon-caret {
 			position: absolute;
-			margin: 0;
 			right: 0;
+			bottom: 50%;
+			margin: 0 0 -10px 0;
```

Comment:
10px is still a magic number.
(and BTW you should not use px, but rem).

Doesn't it work with flexbox ?

Reply:
Yes, you probably deal with that weird situation by using:

```diff
.icon-caret {
  position: absolute;
+  margin: 0;
+ top: 0;
  right: 0;
+ bottom: 0;
-  bottom: 50%;
-  margin: 0 0 -10px 0;
+ display: flex;
+ align-items: center;
+ justify-content: center;
}
```

Reply:
Thanks Guys, I've made using flexbox
even more weird that just adding display: flex; is enough

Reply:
Just adding one more alternative 😄
If you want to avoid flexbox altogether you could also do this:
```
.icon-caret {
  position: absolute;
  right: 0;
  bottom: 50%;
  transform: translateY(50%);
  margin: 0;
}
```